docs/source/conf.py

```diff
@@ -33,35 +33,35 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    'sphinx.ext.autodoc',
-    'sphinx.ext.viewcode',
+    "sphinx.ext.autodoc",
```

Comment:
it looks like `isort` was run directly instead of using `openwisp-qa-format`, because the latter does not convert single quotes to double quotes. We want to avoid doing that change because it pollutes the history to no advantage, it will make it harder for us to review the changes as well

Reply:
Yeah, I had all sorts of problems with this actually. `openwisp-qa-format` and the configurations we have presently have contradictions between `black` and `flake8`, so this is all bit of a nightmare for a new contributor - builds fail, advise running `openwisp-qa-format`. So, you do, then you find it never manages to format code successfully, so you're left using individual tools.

Reply:
Anyway, I'll clean this all up as I get through the work - I just needed to get qa script to pass for now so I could move forward.